packages/core/test/integration/model/sync.test.js

```diff
@@ -1,12 +1,24 @@
 'use strict';
 
 const { expect } = require('chai');
-const { DataTypes, Deferrable, Model } = require('@sequelize/core');
-const { getTestDialect, getTestDialectTeaser, sequelize } = require('../support');
+const {
+  DataTypes,
+  Deferrable,
+  Model,
+  TemporalPeriodType,
+  TemporalTableType,
+} = require('@sequelize/core');
+const {
+  allowDeprecationsInSuite,
+  getTestDialect,
+  getTestDialectTeaser,
+  sequelize,
+} = require('../support');
 
 const dialect = getTestDialect();
 
 describe(getTestDialectTeaser('Model.sync & Sequelize#sync'), () => {
+  allowDeprecationsInSuite(['SEQUELIZE0015']);
```

Comment:
Not sure why this is now required.
The tests would fail otherwise at test `updates the deferrable property of a foreign key` or `should not recreate a foreign key if it already exists when { alter: true } is used with a custom schema`